relm-derive/src/lib.rs

```diff
@@ -81,7 +81,7 @@ fn derive_display_variant(ast: &Item, krate: &Ident) -> TokenStream {
         };
 
         let variant_patterns = enum_item.variants.iter().map(|variant| {
-            let attrs = &variant.attrs;
+            let attrs = variant.attrs.iter().filter(|attr| attr.path.segments[0].ident != "doc");
```

Comment:
I started switching these checks of the first segment to using `is_ident()`. I think it's more correct because it checks there's only one segment as well.
I should be something like:
```rust
.filter(|attr| !attr.path.is_ident(&dummy_ident("doc")))
```
Maybe it would be worth creating an additional function to avoid having to call `dummy_ident()` everytime.

Reply:
Fixed, thank you.